masktoken/masktoken.go

```diff
@@ -21,18 +21,19 @@ import (
 	"regexp"
 )
 
-func MaskTokenFromString(log string, token string) string {
+// MaskTokenFromString redacts a given token from a string. Precisely it replaces it with "*****".
+// It can be used to remove sensitive information from error messages.
```

Comment:
```suggestion
// MaskTokenFromString redacts all matches for the given token from the provided string,
// replacing them with  "*****".
// The token is expected to be [explain details about format/regexp behavior].
// This can for example be used to remove sensitive information from error messages.
```